include/deal.II/dofs/dof_accessor.templates.h

```diff
@@ -2600,6 +2600,10 @@ inline void
 DoFCellAccessor<dimension_, space_dimension_, level_dof_access>::
   get_mg_dof_indices(std::vector<types::global_dof_index> &dof_indices) const
 {
+  Assert(
+    this->dof_handler->mg_vertex_dofs.size() > 0,
+    ExcMessage(
+      "Multigrid DoF indices can only be accessed after DoFHandler::distribute_mg_dofs() has been called!"));
```

Comment:
Or wait, isn't this line longer than 80 characters and should better be broken into two?

Reply:
It's not like we are too consistent in this regard^^ But I'll change it tomorrow.

Reply:
I have split the text up (and also other instances in that file that were too long).